pytest_bdd/utils.py

```diff
@@ -110,3 +110,23 @@ def get_markers_args_using_iter_markers(node, mark_name):
 def get_markers_args_using_get_marker(node, mark_name):
     """Deprecated on pytest>=3.6"""
     return getattr(node.get_marker(mark_name), 'args', ())
+
+
+class ConfigStack(object):
+
+    _stack = []
+
+    @classmethod
+    def add(cls, config):
+        cls._stack.append(config)
+
+    @classmethod
+    def get_previous(cls):
```

Comment:
also because get_previous should not have side effects, but instead it removes the current element and return the previous one

Reply:
Removed